share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -119,12 +129,16 @@ endfunction()
 option(ROCM_SYMLINK_LIBS "Create backwards compatibility symlink for library files." ON)
 
 function(rocm_install_targets)
-    set(options)
+    set(options PRIVATE)
     set(oneValueArgs PREFIX EXPORT COMPONENT)
     set(multiValueArgs TARGETS INCLUDE)
 
     cmake_parse_arguments(PARSE "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
+    if(PARSE_PRIVATE AND ENABLE_ASAN_PACKAGING)
```

Comment:
I want to note as I put this forward that I'm not particularly happy with this implementation, and would greatly appreciate feedback on it.